build/Build.md

```diff
@@ -0,0 +1,192 @@
+* [1.7. Image signing and Verification](#signing)
+
+<br/>
+
+## 1.1. Why should you choose Docker as your container runtime ?
+
+Started just 7 years ago in 2013, Docker introduced what would become the industry standard for containers and is undeniably yet the defacto standard to build, share and run containerized apps. 
+
+<img src="../imgs/build/docker-usage.jpg" width="550">
+
+Among many factors which led WSO2 to choose and recommend Docker as its first choice for a container runtime, following are key.
+
+* Docker’s ability to manage a complete container lifecycle.
+* Less learning curve and advanced feature set.
+* Wide community support towards its development and maintenance.
+* High popularity and customer demand.
+
+<br/>
+<img src="../imgs/build/docker-adoption.png" width="900">
+
+
+## 1.2. Using WSO2 images
+
+WSO2-curated product Docker images are available for usage from both :
+
+[1] [Docker Hub](https://hub.docker.com/u/wso2/) : <br/>
+* Contains images built with the inclusion of GA released product packs and is publicly available for use free of charge.
+
+[2] [WSO2 Private Docker Registry, docker.wso2.com](https://docker.wso2.com/) : <br/>
+* Contains Docker images built with the inclusion of regularly updated product packs and is accessible only for its subscribed customers. 
+
+WSO2 recommends using these images in containerized deployments of WSO2 products. This is mainly because WSO2 has aligned its Docker images with its recommended best practices for product deployments.
+
+Following are some of the key features of WSO2 product Docker images, which aligns them with WSO2-recommended best practices for container based product deployments:
+
+* *WSO2 recommends applying changes to the default configurations of a WSO2 product running inside a container via a volume mount. Our Docker images support this feature via a dedicated volume mount by the name:* ```wso2-config-volume``` <br/> (Refer to [this article](https://medium.com/faun/wso2-products-dockerized-advanced-usage-part-1-ae255ef02661) for more details)
+
+* *WSO2 recommends introducing additional artifacts / non-configuration files (e.g. third-party libraries, Carbon extensions in the form <br/> of OSGi bundles, Carbon Applications or any security-related artifacts such as Java Keystore files) to a WSO2 product <br/>running inside a container via a volume mount. Our Docker images support this feature via a dedicated volume <br/>mount by the name:* ```wso2-artifact-volume``` (Refer to [this article](https://medium.com/faun/wso2-products-dockerized-advanced-usage-part-2-9c851b0db557) for more details)
+
+* *WSO2 recommends introducing patches (or hotfixes) to a WSO2 product running inside a container via a volume mount. <br/>Our Docker images support this feature via a dedicated volume mount by the name:* ```wso2-patch-volume``` <br/>(Refer to [this article](https://medium.com/faun/wso2-products-dockerized-advanced-usage-part-2-9c851b0db557) for more details)
```

Comment:
@DilanUA we no longer maintain this mount point. Shall we please remove it?

Reply:
you mean the patch dir?

Reply:
Nope https://github.com/wso2/docker-apim/issues/317.

Reply:
Removed.